.github/workflows/ci-it.yaml

```diff
@@ -49,3 +49,49 @@ jobs:
           CODECOV_TOKEN="d2065307-8f01-4637-9715-2781ef096db7" bash <(curl -s https://codecov.io/bash)
       - name: 'Check Dependencies Licenses'
         run: tools/dependencies/check-LICENSE.sh
+
+  CI-on-Windows:
+    runs-on: indows-latest
+    timeout-minutes: 180
+    strategy:
+      fail-fast: true
+    steps:
+      - uses: actions/checkout@v1
+        with:
+          submodules: true
+      - uses: actions/cache@v1
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
```

Comment:
Actually we don't need cache, because we use `export MAVEN_OPTS='-Dmaven.repo.local=.m2/repository` when running `mvn`, so it caches nothing, or we should remove `-Dmaven.repo.local=.m2/repository` below

Reply:
Do we remove all caches? They exist on the old CI linux section too.

Reply:
> Do we remove all caches? They exist on the old CI linux section too.

All the caches in CI are unnecessary because we specify another local repositories directory in `MAVEN_OPTS='-Dmaven.repo.local=.m2/repository`, and I suggest to remove caches because we want to rebuild the packages (especially for SkyWalking modules) every time in every PR, in case we bring the packages from cache (in previous build), which may be wrong

Reply:
I think we don't need `-Dmaven.repo.local=.m2/repository` as we are running on the GitHub action, right?

Reply:
> I think we don't need `-Dmaven.repo.local=.m2/repository` as we are running on the GitHub action, right?

Yes for sure, it should be removed when migrating into GitHub Action